packages/gatsby-transformer-documentationjs/src/gatsby-node.js

```diff
@@ -45,7 +45,7 @@ function prepareDescriptionNode(node, markdownStr, name, helpers) {
 exports.sourceNodes = ({ actions }) => {
   const { createTypes } = actions
   const typeDefs = /* GraphQL */ `
-    type DocumentationJs implements Node {
+    type DocumentationJs implements Node @childOf(types: ["File", "DocumentationJs"]) {
```

Comment:
Would anything break if we specify that the file is a child of itself? (each `DocumentationJs` node has child nodes)

Also should we add a `mimeTypes: ["application/javascript"]` here too?

Reply:
I've run a quick test and specifying `childOf` of self doesn't seem to break anything. As for `mimeTypes` - this should also work but I don't enough context on this plugin to say if it makes sense here.